lib/services/network_service.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:http/http.dart' as http;
+import 'dart:convert';
+
+import 'package:marvel/model/character.dart';
+
+const _uri =
+    'https://gateway.marvel.com:443/v1/public/characters?modifiedSince=2016-03-05T13%3A17%3A50-0500&limit=100&ts=2&apikey=c1bba7288e4f2f4f744591622a48412b&hash=bab03858fdeab2fe461725bad8d65904';
+
+class NetworkService {
+  Future<List<Character>?> get() async {
+    final response = await http.get(
```

Comment:
Please use one of the network clients we have learn in the Network module

Reply:
I use http 0.13.4 from pub.dev. It doesn't match?

Reply:
Dio and Chopper are more advance and used in prod - so it would be good to work with them to get better expecrience.